onitu/api/__main__.py

```diff
@@ -30,16 +30,134 @@
 escalator = Escalator(session)
 logger = Logger("REST API")
 
+ONITU_READTHEDOCS = "https://onitu.readthedocs.org/en/latest/api.html"
+HOST = escalator.get(u'api:host') or 'localhost'
```

Comment:
This doesn't work, you must use `escalator.get(u'api:host', default='localhost')`. Returning `None` for get by default was considered as by design, as it hides the exception.